test/runtests.jl

```diff
@@ -274,6 +287,16 @@ end
             @test S269{Int}(2.0).x === 2
             @test S269(2.0).x === 2.0
         end
+
+        @testset "check that we can define custom hash function" begin
+            @auto_hash_equals_cached runtests.myhash struct S275
```

Comment:
niceeee 😍

Reply:
why is the `runtests.` needed here? would just `myhash` work?

Reply:
I am testing a qualified name to make sure it works.

Reply:
I will make sure to test unqualified as well.

Reply:
The qualified name is required inside `@testset` because otherwise a definition of `myhash` would be a local function.